src/appleseed-max-impl/appleseedrenderer/appleseedrendererparamdlg.cpp

```diff
@@ -410,6 +410,7 @@ namespace
                 {
                   case IDC_BUTTON_LOG:
                     m_renderer->show_last_session_log();
+                    break;
```

Comment:
Good catch!

There's a typo in the commit title (*openning* -> *opening*).

Reply:
fixing it and force-pushing new commit.